imap_processing/tests/spice/test_geometry.py

```diff
@@ -122,6 +124,58 @@ def test_get_spin_data():
     }, "Spin data must have the specified fields."
 
 
+@pytest.mark.parametrize(
+    "instrument",
+    [
+        SpiceFrame.IMAP_LO,
+        SpiceFrame.IMAP_HI_45,
+        SpiceFrame.IMAP_HI_90,
+        SpiceFrame.IMAP_ULTRA_45,
+        SpiceFrame.IMAP_ULTRA_90,
+        SpiceFrame.IMAP_SWAPI,
+        SpiceFrame.IMAP_IDEX,
+        SpiceFrame.IMAP_CODICE,
+        SpiceFrame.IMAP_HIT,
+        SpiceFrame.IMAP_SWE,
+        SpiceFrame.IMAP_GLOWS,
+        SpiceFrame.IMAP_MAG,
+    ],
+)
+def test_get_instrument_spin_phase(instrument, fake_spin_data):
+    """Test coverage for get_instrument_spin_phase()"""
+    ets = np.array([7.5, 30, 61, 75, 106, 121, 136])
```

Comment:
can you spell `est` out?

Reply:
Sure... that should be MET instead of ephemeris time (et) anyway.